contrib/debug-rn-examples.md

```diff
@@ -0,0 +1,124 @@
+# How to Debug using React Native Examples
+
+## Steps to link local `realm` and `@realm/react` to examples
+
+### 1. Add example to `workspaces` in `packages.json`
+For example, to add `rn-todo-list` on would add it like so:
+```json
+...
+"workspaces": [
+	"examples/rn-todo-list/frontend/",
+	"packages/realm/bindgen/",
+	"packages/realm/bindgen/vendor/realm-core/",
+	"packages/babel-plugin",
+...
+```
+
+### 2. Update relative paths to `node-modules` in native configurations
+
+#### iOS as of `react-native@0.72.6`
+
+Open the `xcworkspace` in Xcode:
+```
+open ios/*.xcworkspace
+```
+
+Open the build phase settings as shown in the following screen shot:
+
+![Build Phases](./assets/ios-build-phase-settings.png)
+
+
+Edit the highlighted lines with the following:
+
+```bash
+WITH_ENVIRONMENT="../../../../node_modules/react-native/scripts/xcode/with-environment.sh"
+REACT_NATIVE_XCODE="../../../../node_modules/react-native/scripts/react-native-xcode.sh"
+```
+
+####  Android as of `react-native@0.72.6`
+
+The following files will require updates:
+`android/app/build.gradle`
+
+at line 13 uncomment and modify `reactNativeDir`
+```gradle
+     //   The folder where the react-native NPM package is. Default is ../node_modules/react-native
+-    // reactNativeDir = file("../node_modules/react-native")
++    reactNativeDir = file("../../../../../node_modules/react-native")
+```
+
+and update line 123
+
+```gradle
+-apply from: file("../../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesAppBuildGradle(project)
++apply from: file("../../../../../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesAppBuildGradle(project)
+```
+
+`android/settings.gradle`
+
+lines 2 and 4 should be updated
+
+```gradle
+-apply from: file("../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesSettingsGradle(settings)
++apply from: file("../../../../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesSettingsGradle(settings)
+ include ':app'
+-includeBuild('../node_modules/@react-native/gradle-plugin')
++includeBuild('../../../../node_modules/@react-native/gradle-plugin')
+```
+
+### 3. Apply our custom `metro` config
+We have a custom `metro` configuration which will help find any linked dependencies in the mono-repo.  First add it to the `devDependencies` in the projects `package.json`:
```

Comment:
Just adding some distinction between the Realm "project" and the Example "project" by just using "example" instead. Perhaps this clarifies it.
```suggestion
We have a custom `metro` configuration which will help find any linked dependencies in the mono-repo.  First add it to the `devDependencies` in the example's `package.json`:
```